e2e/node/resource.test.ts

```diff
@@ -359,18 +317,15 @@ describe("Authenticated end-to-end", () => {
 
   it("can customize the fetch to get and set HTTP headers", async () => {
     let headers: Headers = new Headers();
-    const customFetch: typeof fetch = async (
-      info: Parameters<typeof fetch>[0],
-      init?: Parameters<typeof fetch>[1],
-    ) => {
-      const response = await fetchOptions.fetch(info, {
-        ...init,
+    const customFetch: typeof fetch = async (...args) => {
+      const response = await fetchOptions.fetch(args[0], {
+        ...args[1],
         headers: {
-          ...init?.headers,
+          ...args[1]?.headers,
           "User-Agent": "some-user-agent",
         },
       });
-      if (info.toString() === sessionResource) {
+      if (args[0].toString() === sessionResource) {
```

Comment:
Why this change? I find the previous version to be more readable.

Reply:
My editor was complaining (see below) but this doesn't actually seem to be an issue in CI so I've reverted the changes.

![image](https://github.com/inrupt/solid-client-js/assets/63333554/fa17ac76-f6b2-4702-9bf6-55a6ce349f8b)